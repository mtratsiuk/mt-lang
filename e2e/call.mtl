(print (+ 1 -2))

(print
  (-
    1
    2))

(console/log)

(print (+ 2 2))
(print (= !true false))
(print (and true false))
(print (or true false))
(print (< 10 5))
(print (>= 10 5))
(print (<= 10 5))
