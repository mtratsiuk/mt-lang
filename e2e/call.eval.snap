-1
-1

4
true
false
true
false
true
false
